sidebars.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
       type: 'category',
       label: 'Home',
       items: [
+        'release-notes-v4',
```

Comment:
You to create a new sub section at the beginning and add version 3 and version 4 there instead